content/docs/user-guide/interactivity/robotics/vehicle-dynamics.md

```diff
@@ -0,0 +1,78 @@
+| `DriveModel / Steering PID / IMax`             | Maximum integration impact of PID.                                       |
+| `DriveModel / Steering PID / AntiWindUp`       | Prevents integral wind-up in PID.                                        |
+| `DriveModel / Steering PID / OutputLimit`      | Clamps output to maximum value.                                          |
+| `DriveModel / Vehicles Limits / Speed limit `  | Maximum achievable linear speed in meters per second.                    |
+| `DriveModel / Vehicles Limits / Steering limit`| Maximum achievable steering angle.                                       |
+
+### Skid Steering Drive Model
+The model computes velocities in the joints of the vehicle and applies it accordingly to commanded velocity and configuration.
+
+![SkidSteeringModel](/images/user-guide/gems/ros2/skidSteeringModel.png)  
+Parameters of the model are exposed to the user via `AckermannVehicleModelComponent`:
+
+| Parameter Name                                       | Description                                                        |
+|------------------------------------------------------|--------------------------------------------------------------------|
+| `DriveModel / Axles `                                | List of axles of the vehicle.                                      |
+| `DriveModel / Axles / Axle Wheels `                  | List of wheels in axis.                                            |
+| `DriveModel / Axles / Is it a steering`              | It is ignored in this model.                                       |
+| `DriveModel / Axles / Is it a drive`                 | If it is enabled, the Skid Steering Drive Model will apply velocities to the axis' wheels.|
+| `DriveModel / Axles / Track`                         | Distance between front and rear axis.                              |
+| `DriveModel / Axles / Wheelbase`                     | Distance between left and right wheel.                             |
+| `DriveModel / Vehicles Limits / Linear speed limit ` | Maximum achievable linear speed in meters per second.              |
+| `DriveModel / Vehicles Limits / Angular speed limit` | Maximum achievable angular speed in radians per second.            |
+
+### Manual control
+
+The `VehicleModel` will handle input events with names "steering" and "accelerate". This means you can add an [InputComponent](/docs/user-guide/components/reference/gameplay/input/) to the same entity and define an input map for your input devices (such as keyboard or a game pad) to control the vehicle manually.
+
+You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/) to move your robot with Twist messages. `RobotControl` is suitable to use with [ROS 2 navigation stack](https://navigation.ros.org/).
+
+It is possible to implement your own control mechanisms with this Component.
```

Comment:
```suggestion
It is possible to implement your own control mechanisms with this component.
```
according to contributing/to-docs/style-guide/format/#bold-gems-and-components
'Use lowercase and unformatted text for the word “component”.'